SmartApi/smartConnect.py

```diff
@@ -433,6 +435,27 @@ def searchScrip(self, exchange, searchscrip):
         else:
             return searchScripResult
         
+    def ind_order_details(self, qParam):
+        url = self._rootUrl + self._routes["api.individual.order.details"] + qParam
+        headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+            "X-UserType": "USER",
+            "X-SourceID": "WEB",
+            "X-PrivateKey": self.api_key,
+        }
+
+        if self.access_token:
+            headers["Authorization"] = "Bearer " + self.access_token
```

Comment:
This access_token processing is also same. use common method

Reply:
created common method